_run/kube/deployment.yaml

```diff
@@ -5,7 +5,7 @@ services:
   web:
     image: quay.io/ovrclk/demo-app
     expose:
-      - port: 80
+      - port: 8080
```

Comment:
Do we still need this?

Reply:
I thought I had removed this change, I'll double check.